src/zenml/zen_server/routers/run_metadata_endpoints.py

```diff
@@ -61,3 +63,29 @@ def list_run_metadata(
     return zen_store().list_run_metadata(
         run_metadata_filter_model, hydrate=hydrate
     )
+
+
+@router.get(
+    "/{run_metadata_id}",
+    response_model=RunMetadataResponse,
+    responses={401: error_response, 404: error_response, 422: error_response},
+)
+@handle_exceptions
+def get_run_metadata(
```

Comment:
This endpoint seems pretty useless to me since you usually always want to get all metadata related to some entity, not a single entry in the DB. Does this have any usage in code? If not, I'd suggest we get rid of it

Reply:
Didn't see this used anywhere during code review, so would indeed suggest to delete this

Reply:
It was missing for REST, which existed for the SQL Store. In `RunMetadataResponse` it is called in `get_hydrated_version`, which, I assume, can happen over REST as well. @bcdurak , WDYT?

Reply:
Ah I see, yes then we need this endpoint now, makes sense 👍